src/reducers/index.js

```diff
@@ -51,43 +51,29 @@ export default (state = DEFAULT_STATE, action) => {
       });
     }
     case SET_VEGA_EXAMPLE:
-      try {
```

Comment:
I removed the try catch because SET_VEGA_EXAMPLE or SET_VEGA_LITE_EXAMPLE shouldn't have any errors or warnings